include/zephyr/sys/spsc_pbuf.h

```diff
@@ -31,11 +31,25 @@ extern "C" {
 #define SPSC_PBUF_UTILIZATION_OFFSET 8
 
 /**@} */
+#ifdef DCACHE
+#ifdef CONFIG_DCACHE_LINE_SIZE
+#define Z_SPSC_PBUF_LOCAL_DCACHE_LINE CONFIG_DCACHE_LINE_SIZE
+#elif !defined(CONFIG_DCACHE_LINE_SIZE_DETECT)
+#define Z_SPSC_PBUF_LOCAL_DCACHE_LINE DT_PROP_OR(CPU, d_cache_line_size, 0)
+#else
+#define Z_SPSC_PBUF_LOCAL_DCACHE_LINE 0
```

Comment:
You can merge this two cases (if the DT property doesn't exist, it falls back to 0)

Reply:
merged